test.js

```diff
@@ -5,7 +5,8 @@ const { performance } = require('perf_hooks')
 const { Transform } = require('stream')
 const Fastify = require('fastify')
 const Stats = require('.')
-const fakeTimer = require('@sinonjs/fake-timers')
+// const fakeTimer = require('@sinonjs/fake-timers')
```

Comment:
is this intentional?

Reply:
Yes due to the lining.

I turned off this module due this issue: https://github.com/sinonjs/fake-timers/issues/430
I can create a proper commit to revert instead of comment the source code